tools/ITER_Profile_reader.py

```diff
@@ -0,0 +1,66 @@
+# This class provides routines for reading input profiles from CORSICA data files
+
+import numpy as np
+
+class ITER_Profile_reader:
+
+    def __init__(self, filename):
+        self.filename = filename
+        
+        f = open(self.filename,'r')
+        lines = f.readlines()
+        iLine = 0
+        keep_reading = True
+        
+        # Go through the file and find basic scalar data 
+        # (number of columns ncol for the profile datasets, 
+        # number of radial gridpoints N_Rho and the plasma current Ip)
+        while keep_reading:
+            line = lines[iLine]
+            #print(line)
+            if line.split():
+                if line.split()[0] == 'ncol':
```

Comment:
``line.split()`` creates an array. Wouldn't it better to store that array once in each ``while`` iteration, instead of calling ``line.split()`` multiple times?